.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: martinthomson/i-d-template:latest
+    working_directory: ~/draft
+
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "03:14:d6:bb:0b:f5:d7:4c:59:d1:2b:33:13:75:01:7a"
```

Comment:
This step isn't in the base version of this file.  I suspect that it won't do anything here.

Reply:
afaik if I don't add this key, circle won't be able to commit on the gh-branch (at least in my trees) as it uses the read-only key.

Reply:
Probably if I remove the read-only key reference from CI it would be fine.

Reply:
afaik circleci uses the default, read-only, key without this. Probably  if you remove the read-only key it is not needed.